lib/payload/scanner/service/engine.py

```diff
@@ -1,287 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I think you removed the whole file by mistake!

Reply:
yes i did but I added it back again.